core/AppInfo/ConfigLexicon.php

```diff
@@ -49,6 +52,16 @@ public function getAppConfigs(): array {
 				lazy: true,
 				note: 'Shares with guessable tokens may be accessed easily. Shares with custom tokens will continue to be accessible after this setting has been disabled.',
 			),
+			new Entry(self::SHARE_LINK_PASSWORD_DEFAULT, ValueType::BOOL, false, 'Always ask for a password when sharing document'),
```

Comment:
```suggestion
			new Entry(self::SHARE_LINK_PASSWORD_DEFAULT, ValueType::BOOL, false, 'Ask for a password when sharing document by default'),
```
Unless the option name is broken, this is not always but a default?

Reply:
I guess the logic here is "always ask to set a password when sharing, but it may not be filled" or something like that? :thinking: 
@kra-mo What do you think, does the wording make sense?